index.html

```diff
@@ -345,6 +217,60 @@ <h2 class="contact-title">Contact me</h2>
 
     </div>
 
+<!-- POPUP WINDOW DETAILS -->
+<div class="popup-details" id="details-modal">
+  <div class="details-content">
+    <div class="popup-header">
+      <h2 id="project-name">Multi-Post Stories</h2>
+      <img
+        id="close-popup"
+        src="./assets/close-popup.png"
+        alt="Close pupup button icon"
+      />
+    </div>
+    
+    <ul class="s-desc" id="popup-ul-tags">
+      <li>CANOPY</li>
+      <li><div class="bullet"></div></li>
+      <li>Bac End Dev</li>
+      <li><div class="bullet"></div></li>
+      <li>2015</li>
+    </ul>
+    
+      <img
+        id="popup-screenshot"
+        class="screenshot"
+        src="./assets/tonic-nature.png"
+        alt="project image"
+      />
+    <div class="popup-desc">
+      <p class="popup-text-desc" id="popup-desc">
+        Lorem Ipsum is simply dummy text of the printing and typesetting
+        industry. Lorem Ipsum has been the industry's standard dummy text
+        ever since the 1500s, when an unknown printer took a galley of type
+        and scrambled it to make a type specimen book. It has survived not
+        only five centuries, but also the leap into electronic typesetting,
+        remaining essent
+      </p>
+      <div class="popup-footer">
+        <ul class="categories fs"  id="popup-ul-techs">
+          <li>html</li>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <hr class="line">
+        <div class="links-popup">
+          <a class="btn-popup" href="#" target="_blank" rel="noopenner" id="see-live">See Live <img class="btn-icon" src="./assets/see-live.png" alt="Github blue icon" ></a>
+          <a class="btn-popup" href="#" target="_blank" rel="noopenner" id="project-source">See Source <img class="btn-icon" src="./assets/github-blue.png" alt="See Live blue icon" ></a>
+        </div>
+      </div>
+    </div>
```

Comment:
- This is what the project requirements ask for:
![image](https://user-images.githubusercontent.com/83596135/189400760-cec4f52c-ab6e-4962-b57a-e6f1f1ee0b49.png)

- [ ] In this particular project, the section of the whole project along with the models should be created dynamically with Javascript. I can see that the works section data is with the data from the `projects` array good job! 👏🏻 , but the Modal popup is still not being created with Javascript.  Please use Javascript to dynamically render your Modals or Popup windows pages as you did with the WORKS PROJECT SECTION.

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `projects` array. 😉 👍🏻 
 

After you have done this, please ask for another review in your dashboard.